### PR TITLE
[IntraHealth] New dashboards

### PR DESCRIPTION
**Pull request by panprosiak**
New custom reports for IntraHealth

Reports with calculations and UI (test data):
-> Disponibilite
-> Taux de Rupture
-> Consommation
-> Recap Passage 1

Reports with calculations and UI (fake data):
-> Taux de Perte
-> Taux de Peremption
-> Taux de Satisfaction
-> Indicateurs de Base

Reports without calculations and UI (just filters):
-> Valeur des Stocks PNA Disponible
-> Recap Passafe 2

### CODE REVIEW

File: custom/intrahealth/reports/indicateurs_de_base_dasboard.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+from corehq.apps.locations.models import SQLLocation
+from corehq.apps.reports.datatables import DataTablesHeader, DataTablesColumn
+from corehq.apps.reports.standard import CustomProjectReport
+from custom.intrahealth.filters import YeksiNaaLocationFilter, FRMonthFilter, FRYearFilter
+from custom.intrahealth.reports.utils import YeksiNaaMonthYearMixin
+
+
+class IndicateursDeBaseReport(CustomProjectReport, YeksiNaaMonthYearMixin):
+    # TODO: we need new filters and Data
```
Comment (gbova) [inline custom/intrahealth/reports/indicateurs_de_base_dasboard.py]: There are lots of `TODO` comments. Can we do them or get rid of them?

**Review by gbova — APPROVED** [review]
I have some comments, nothing blocking. As far as I can tell, this looks pretty good. And I know we need to get this out for testing ASAP, so I think we should be good to

**Comment by MaciejChoromanski** [comment]
Thanks for the helpful comments. We'll do a cleanup right away and we'll refactor our code when all of the dashboards are done.

FYI @dborowiecki @tstalka

**Comment by MaciejChoromanski** [comment]
@gbova 
I've changed fragments of code mentioned in comments. Also, now 'Taux de Satisfaction' report uses test data instead of fake data.

**Pull request merged**